lib/apress/selenium_eti/spec/company_site/eti/product_price_spec.rb

```diff
@@ -39,88 +42,122 @@
 
     context 'когда цена от и до', story: 'когда цена от и до' do
       context 'когда только "от"' do
-        before do
-          @price_from = {from: Faker::Number.number(digits: 2)}
-          @cs_eti_page.set_price_from_to(@price_from)
-          @cs_eti_page.wait_saving
-          @cs_eti_page.search_product(@name)
+        let(:price) do
+          {
+            type: :range, 
+            price: Faker::Number.number(digits: 2)
+          }
         end
 
         it 'введеная цена отображается' do
-          if @cs_eti_page.project_pulscen?
-            expect(@cs_eti_page.product_price(@name)).to eq 'от ' + @price_from[:from].to_s + ' руб.'
-          elsif @cs_eti_page.project_blizko?
-            expect(@cs_eti_page.product_price(@name)).to eq 'от ' + @price_from[:from].to_s + ' ₽'
+          if @cs_eti_table.project_pulscen?
+            expect(@cs_eti_table_products.price(@product)).to eq("от #{price[:price]} руб.")
+          elsif @cs_eti_table.project_blizko?
+            expect(@cs_eti_table_products.price(@product)).to eq("от #{price[:price]} ₽")
           else
             raise ArgumentError, 'locator not found'
           end
         end
       end
 
       context 'когда только "до"' do
-        before do
-          @price_to = {to: Faker::Number.number(digits: 2)}
-          @cs_eti_page.set_price_from_to(@price_to)
-          @cs_eti_page.wait_saving
-          @cs_eti_page.search_product(@name)
+        let(:price) do
+          {
+            type: :range, 
+            price_max: Faker::Number.number(digits: 2)
```

Comment:
Style/TrailingCommaInHashLiteral: Put a comma after the last item of a multiline hash.